core/src/main/java/org/apache/accumulo/core/fate/zookeeper/ZooReader.java

```diff
@@ -44,7 +43,7 @@ public class ZooReader {
 
   protected static final RetryFactory RETRY_FACTORY =
       Retry.builder().maxRetries(10).retryAfter(250, MILLISECONDS).incrementBy(250, MILLISECONDS)
-          .maxWait(5, SECONDS).backOffFactor(1.5).logInterval(3, MINUTES).createFactory();
+          .maxWait(2, MINUTES).backOffFactor(1.5).logInterval(3, MINUTES).createFactory();
```

Comment:
Just to be clear, this is the max time to wait between retries, not the maximum time to wait for all retries to finish. What are the risks with changing this so dramatically?

Reply:
For reference, with a backoff factor of 1.5, we go from 250ms to 5 seconds in about 8 rounds, and to 2 minutes in about 16 rounds. If we want it to take longer to reach the upper limit, we could reduce the backoff factor with this change. If we change it to 1.3, it will take about 12 rounds to reach 5 seconds and 24 rounds to reach 2 minutes.

Reply:
Yeah this is the max wait time that it will grow to.  Starts off sleeping 250 ms and then grows from there until getting to 2 minutes if it keeps failing.

One risk is  that single/few process that are failing when no other process is failing will not be as responsive as before.  

The reason I am thinking of increasing the max is for the thundering heard case.  Like if there are 4K tservers all trying to mutate the same ZK node, backing off to 2 min would have on average 33 tservers per second trying to mutate when reaching max wait time.

Reply:
Well, I'm okay with these changes. Not sure what is optimal for the backoff factor with 2 minutes as the max wait, though.

Reply:
Could also lower the increment amount instead of the backoff factor. It doesn't need to be 250ms. It could be 100ms with 1.5 backoff, or even 1.3 backoff.

Reply:
yeah decreasing the increment and increasing the max wait makes a lot sense.  It will normally be responsive even when retrying a few times, and will only become less responsive when it has retried a lot which is acceptable.

Reply:
One problem to consider when tweaking this, though is that it is also set to a max number of retries of 10... so with 100ms intervals and 1.5 backoff factor, you're maxing out around 5 seconds by the time it reaches the 10th retry anyway. This won't affect the case where one of the retries is not decremented, because it's an "always retry" case, like on #3720, but it might be worth tweaking the max number of retries as well for the other cases.

Reply:
good point, I was only thinking about the retry forever case.

Reply:
I put the increment back to 250ms so that the retries spread over a longer time like they used to be for the limited retry case